scripts/upower/95-upower-hid.hwdb

```diff
@@ -0,0 +1,179 @@
+##############################################################################################################
+# Uninterruptible Power Supplies with USB HID interfaces
+#
+# This file was automatically generated by NUT:
+# https://github.com/networkupstools/nut/
+#
+# To keep up to date, monitor upstream NUT
+# https://github.com/networkupstools/nut/commits/master/scripts/upower/95-upower-hid.hwdb
+# or checkout the NUT repository and call 'tools/nut-usbinfo.pl'
+
+usb:v03f0p*
+ UPOWER_VENDOR=Hewlett Packard
+
+usb:v0463p*
+ UPOWER_VENDOR=Eaton
+
+usb:v047cp*
+ UPOWER_VENDOR=Dell
+
+usb:v0483p*
+ UPOWER_VENDOR=ST Microelectronics
+
+usb:v04b3p*
+ UPOWER_VENDOR=IBM
+
+usb:v04d8p*
+ UPOWER_VENDOR=Minibox
+
+usb:v050dp*
+ UPOWER_VENDOR=Belkin
+
+usb:v051dp*
+ UPOWER_VENDOR=APC
+
+usb:v0592p*
+ UPOWER_VENDOR=Powerware
+
+usb:v05ddp*
+ UPOWER_VENDOR=Delta UPS
+
+usb:v06dap*
+ UPOWER_VENDOR=Phoenixtec Power Co., Ltd
+
+usb:v075dp*
+ UPOWER_VENDOR=iDowell
+
+usb:v0764p*
+ UPOWER_VENDOR=Cyber Power Systems
+
+usb:v09aep*
+ UPOWER_VENDOR=TrippLite
+
+usb:v0d9fp*
+ UPOWER_VENDOR=PowerCOM
+
+usb:v10afp*
+ UPOWER_VENDOR=Liebert
+
+usb:v1cb0p*
+ UPOWER_VENDOR=Legrand
+
+usb:v2341p*
+ UPOWER_VENDOR=Arduino
+
+usb:v2A03p*
+ UPOWER_VENDOR=Arduino
+
+usb:v2b2dp*
+ UPOWER_VENDOR=AEG
+
+usb:v2e51p*
+ UPOWER_VENDOR=Ever
+
+usb:v2e66p*
+ UPOWER_VENDOR=Salicru
+
+usb:v4234p*
+ UPOWER_VENDOR=Powervar
+
+usb:v03f0p0001*
```

Comment:
Is there a particular reason to not include a comment (or `UPOWER_VENDOR` or something) to keep info about UPS makers/sellers here?

The original rules file had blocks with a top comment ("Tripplite" etc.) for groups of IDs, and here is just a wall of text with less detail. Would be great if it can be made functionally equivalent to the old approach.

Reply:
Could you please add that then? :)

Reply:
OK, how about this? It limits setting the vendor on actual UPS devices, which is sufficient. I only did that because it seems quite readable this way.

Reply:
Looks better, posted more questions to clarify specifics :)